module/User/src/User/Model/User.php

```diff
@@ -140,13 +139,13 @@ public function getMember()
      */
     public function getRoleNames()
     {
-        $roles = [];
+        $names = [];
 
         foreach ($this->getRoles() as $role) {
-            $roles[] = $role->getRole();
+            $names[] = $role->getRole();
         }
 
-        return $roles;
+        return $names;
     }
```

Comment:
This isn't incorrect, but I don't think we will benefit from changing this.

Reply:
names in this method shadowed a variable in the class it is contained in